src/caret_analyze/architecture/architecture_exporter.py

```diff
@@ -80,13 +80,20 @@ def _to_dict(self, path_value: PathStructValue):
         obj['path_name'] = path_value.path_name
         node_chain = []
         for node_path in path_value.node_paths:
-            node_chain.append(
-                {
-                    'node_name': node_path.node_name,
-                    'publish_topic_name': node_path.publish_topic_name or UNDEFINED_STR,
-                    'subscribe_topic_name': node_path.subscribe_topic_name or UNDEFINED_STR
-                }
-            )
+            d = {
+                'node_name': node_path.node_name,
+                'publish_topic_name': node_path.publish_topic_name or UNDEFINED_STR,
+                'subscribe_topic_name': node_path.subscribe_topic_name or UNDEFINED_STR
+            }
+
+            if 0 < (node_path.publisher_construction_order or 0):
```

Comment:
Is there any reason to write as `if 0 < (node_path.publisher_construction_order or 0):` insted of `if 0 < node_path.publisher_construction_order:`?